restconnector/RequestHandler.cpp

```diff
@@ -83,7 +95,6 @@ void RequestHandler::processGET_( const ::zeq::Event& event )
 
     while( blocked_ )
     {
-        while( subscriber_->receive( 0 ) ){}
```

Comment:
@chevtche I think you tried to mean: while( subscriber_->receive() ) {}
Note infinite timeout vs 0 timeout.